### ISSUE TITLE
Can't uninstall package

### ISSUE THREAD

**Issue by rgoubet** [flathub/io.github.peazip.PeaZip#25]
After installing the package with `flatpak install flathub io.github.peazip.PeaZip`, I'm unable to uninstall it:

```bash
~$ flatpak uninstall io.github.peazip.PeaZip
**
flatpak:ERROR:common/flatpak-transaction.c:2971:mark_op_resolved: assertion failed: (commit != NULL)
Bail out! flatpak:ERROR:common/flatpak-transaction.c:2971:mark_op_resolved: assertion failed: (commit != NULL)
Abandon (core dumped)
```

I ran `flatpak repair flathub io.github.peazip.PeaZip`, but no better luck.

**Comment by catsout** [comment]
Seems a flatpak bug.  
Better to raise a issue in flatpak's repo.  
https://github.com/flatpak/flatpak

**Comment by wjt** [comment]
https://github.com/flatpak/flatpak/issues/3943 matches this report, is purportedly fixed in https://github.com/flatpak/flatpak/commit/ce6fdcf0778fa9d95680d6c6645517279bcd39ed, which was first released in Flatpak 1.11.2.

**Comment by rgoubet** [comment]
Eventually, I had a notification from the Ubuntu Software Centre that the software was updated, then I was able to uninstall.
